containers/kubernetes/api-cli/using-load-balancer-annotations.mdx

```diff
@@ -242,7 +242,7 @@ The default value is `10m`. The duration is measured by Golang. Duration (ex: `1
 ## service.beta.kubernetes.io/scw-loadbalancer-timeout-connect
 
 This is the annotation to set the maximum initial server connection establishment time.
-The default value is `10m`. The duration are go's time. Duration (ex: `1s`, `2m`, `4h`, ...).
+The default value is `10s`. The duration are go's time. Duration (ex: `1s`, `2m`, `4h`, ...).
```

Comment:
In the [code](https://github.com/scaleway/scaleway-cloud-controller-manager/blob/2f70ff74f90b6b8240da6ded9bf61ae87884974a/scaleway/loadbalancers.go#L1466) this is set to 10m.
The comment at [L132](https://github.com/scaleway/scaleway-cloud-controller-manager/blob/2f70ff74f90b6b8240da6ded9bf61ae87884974a/scaleway/loadbalancers.go#LL132C6-L132C6) is misleading, we'll fix this.